x-pack/auditbeat/module/system/host/users_unix.go

```diff
@@ -0,0 +1,65 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+// +build !windows
```

Comment:
This should also a have a `cgo` build tag (like `!windows,cgo`). I haven't tried this in a while with auditbeat but it's nice if you can still build the project without cgo (CGO_ENABLED=false) even it it has limited features.